### DISCUSSION TITLE
Filter with An Accessor

### DISCUSSION THREAD

**Discussion by ahmetkocabiyik** [filamentphp/filament#9930] [Help]
I want to use table filter using an accessor in model. Filter query method is directly filter from database like below. Is there any way to use an acessor defined in model to filter something in that resource.

```php
Filter::make('is_featured')
    ->query(fn (Builder $query): Builder => $query->where('is_featured', true))
```

**Comment by leandrocfe** [comment]
You can't do it. Maybe your accessor could be a [virtual column](https://filamentphp.com/docs/3.x/forms/fields/select#customizing-the-relationship-option-labels)
